### ISSUE TITLE
can you share simple example to see this mentioned overfit

### ISSUE THREAD

**Issue by Sandy4321** [cod3licious/autofeat#8]
can you share simple example to see this mentioned overfit
Please keep in mind that since the AutoFeatRegressor and AutoFeatClassifier models can generate very complex features, it will likely overfit on noise in the dataset, though the coefficients for features related to noise should be fairly small. It is generally suggested to carefully inspect the features found by autofeat and use those that make sense to you to train your own models

**Comment by cod3licious** [comment]
I tried to clarify this in the readme. Overfitting is only a problem if you have very few samples (and a lot of variety in your dataset). Just watch for the performance on training and test data and if the model is much better on the training than the test set, look at all the features and try to manually delete some that seem too specific to you.
